vms/platformvm/vm_test.go

```diff
@@ -721,9 +721,9 @@ func TestRewardValidatorAccept(t *testing.T) {
 	// Fast forward clock to time for genesis validators to leave
 	vm.clock.Set(defaultValidateEndTime)
 
-	blk, err := vm.Builder.BuildBlock(context.Background()) // should advance time
```

Comment:
PreBanff we used ApricotProposalBlocks to advance time.
Post Banff cleanup, we don't build ApricotProposalBlocks anymore (we deleted the code to do that).
Chain time is advanced by blocks timestamp, without proposal.
This test claims to issue an proposalBlock that advances time; it is instead issuing a proposal block to reward the first validator. Error is not caught because we didn't have a assertion on the block tx type.
Fix is to skip this "advance time with proposal block" section and move directly to next section where genesis validator is rewarded.